docs/src/content/docs/tr/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,286 @@
+---
+title: CSS & Stiller
+description: Starlight sitenizi özel CSS ile nasıl stilize edebileceğinizi veya Tailwind CSS ile nasıl entegre edebileceğinizi öğrenin.
+---
+
+Starlight sitenizi özel CSS dosyaları ile stilize edebilir veya Starlight Tailwind eklentisini kullanabilirsiniz.
+
+## Özel CSS Stilleri
+
+Starlight sitesine uygulanan stilleri özelleştirmek için, Starlight'ın varsayılan stillerini değiştirmek veya genişletmek için ek CSS dosyaları sağlayarak stil özelleştirebilirsiniz.
+
+1. `src/` dizinine bir CSS dosyası ekleyin. 
+	Örneğin, varsayılan sütun genişliğini artırabilir ve sayfa başlıkları için daha büyük metin boyutu ayarlayabilirsiniz:
+
+   ```css
+   /* src/styles/custom.css */
+   :root {
+   	--sl-content-width: 50rem;
+   	--sl-text-5xl: 3.5rem;
+   }
+   ```
+
+2. CSS dosyanızın yolunu `astro.config.mjs` içindeki Starlight'ın `customCss` dizisine ekleyin:
+
+   ```js
+   // astro.config.mjs
+   import { defineConfig } from 'astro/config';
+   import starlight from '@astrojs/starlight';
+
+   export default defineConfig({
+   	integrations: [
+   		starlight({
+   			title: 'Docs With Custom CSS',
+   			customCss: [
+   				// Özel CSS dosyanızın göreli yolunu.
+   				'./src/styles/custom.css',
+   			],
+   		}),
+   	],
+   });
+   ```
+
+Starlight tarafından kullanılan ve sitenizi özelleştirmek için ayarlayabileceğiniz tüm CSS özel özellikleri, [GitHub'daki `props.css`](https://github.com/withastro/starlight/blob/main/packages/starlight/style/props.css) dosyasında görüntüleyebilirsiniz.
+
+## Tailwind CSS
+
+Astro projelerinde Tailwind CSS desteği, [Astro Tailwind entegrasyonu](https://docs.astro.build/en/guides/integrations-guide/tailwind/) ile sağlanır. Starlight, Starlight stilleriyle uyumlu olacak şekilde Tailwind'i yapılandırmaya yardımcı olan bir Tamamlayıcı Tailwind eklentisi sunar.
+
+Starlight Tailwind eklentisi aşağıdaki yapılandırmayı uygular:
+
+- Tailwind'in `dark:` varyantlarını Starlight'ın karanlık moduyla uyumlu hale getirir.
+- Starlight'ın kullanıcı arayüzünde Tailwind [tema renklerini ve yazı tiplerini](#styling-starlight-with-tailwind) kullanır.
```

Comment:
I don’t think the links have been updated as needed. For example this one should be:

```suggestion
- Starlight'ın kullanıcı arayüzünde Tailwind [tema renklerini ve yazı tiplerini](#starlightı-tailwind-ile-stilize-etme) kullanır.
```

Could you please check through all my comments in https://github.com/withastro/starlight/pull/1038? It would be a lot of work for me to have to make them again.